src/templates/Footer/index.js

```diff
@@ -31,24 +31,25 @@ function Footer(props) {
   const {
     className,
     phoneNumber,
-    text,
+    emailAddress,
```

Comment:
> explicit exposed email addresses on a site thats search engine crawl-able will always render that mailbox useless with the sheer amount of spam.

Has this been an issue? I don't mind changing it, but I'm curious why. We have our email exposed in multiple places in the app (including in the existing Footer).

> If we allow one or the other or both the onClickChat prop should either be removed or made to work.

Can you please clarify? `onClickChat` works now if passed. If it isn't passed, the button doesn't render.